actionview/lib/action_view/helpers/form_helper.rb

```diff
@@ -1662,7 +1670,7 @@ class FormBuilder
         :number_field, :range_field
       ]
 
-      attr_accessor :object_name, :object, :options
+      attr_accessor :object_name, :object, :errors, :options
```

Comment:
I understand that the motivating use case is to use `field_error_proc` for calls with `scope:` and without `model:`, but if `form.errors` is meant to be public, it also opens the door for in-partial access:

```erb
<%= form_with scope: :password, errors: current_user.errors do |form| %>
  <%= form.label :password %>
  <%= form.password_field :password %>
  <%= form.errors.messages_for(:password).to_sentence if form.errors.include?(:password) %>
<% end %>
```

Is this intentionally a public accessor? If so, is there a way to add documentation for it?

Reply:
> Is this intentionally a public accessor? If so, is there a way to add documentation for it?

Yes, I would like to support the use case you mentioned!  That said, I was just planning to list `errors` alongside `object` under **Attributes** in the [`FormBuilder` API docs](https://api.rubyonrails.org/classes/ActionView/Helpers/FormBuilder.html).  Is there a place in the docs where you would like to see it brought up?

Reply:
How do we document the fact that the value passed to `form_with model: ...` is accessible as `form.object`?

Maybe under the `form_with` method documentation, close to the `errors:` key? I think a brief mention that the value is accessible as `form.errors`.

Reply:
> How do we document the fact that the value passed to `form_with model: ...` is accessible as `form.object`?

From a cursory search, we currently don't, other than the **Attributes** section I mentioned.

> Maybe under the `form_with` method documentation, close to the `errors:` key? I think a brief mention that the value is accessible as `form.errors`.

I like that idea!  Since I was already touching the `:model` option documentation, I've added a mention of `form.object` there too.